TripKit/Classes/helpers/TKUserProfileHelper.swift

```diff
@@ -70,16 +76,39 @@ public class TKUserProfileHelper: NSObject {
   
   @objc public class func setModeIdentifier(_ modeIdentifier: Identifier, toHidden hidden: Bool) {
     update(hiddenModeIdentifiers, forKey: .hidden, modeIdentifier: modeIdentifier, include: hidden)
+    
+    // We give special consideration when modifying walking or wheelchair mode since we may
+    // need to perform toggle, e.g., adding walking while wheelchair is already included
+    // calls for toggle.
+    let isModifyingWalking = modeIdentifier == SVKTransportModeIdentifierWalking
+    let walkingIsCurrentlyIncluded = !modeIdentifierIsHidden(SVKTransportModeIdentifierWalking)
+    let isModifyingWheelchair = modeIdentifier == SVKTransportModeIdentifierWheelchair
```

Comment:
I believe there’s already helper methods somewhere to check is a mode identifier is walking or wheelchair.

Reply:
Those helper methods are in TripGoAppKit, not accessible to TripKit?

Reply:
They are in `SVKTransportModes`

Reply:
remembered why I didn't use those helpers. the `modeIdentifierIsWalking` only tests the prefix `wa_`, so it'd also return true for wheelchair.

Reply:
This has since been removed.